src/spring/azext_spring/tests/latest/test_asa_app_scenario.py

```diff
@@ -288,4 +288,26 @@ def test_vnet_public_endpoint(self):
 
         self.cmd('spring app update -n {app} -g {rg} -s {serviceName} --assign-public-endpoint true', checks=[
             self.check('properties.vnetAddons.publicEndpoint', True)
-        ])
\ No newline at end of file
+        ])
+
+
+@record_only()
```

Comment:
May I ask why this test marked as `record_only`?

Reply:
the scenario depends on the creation of Azure Spring Apps, which is time-consuming and might be failed caused by backend or lack of resources. So I marked it as record_only to using the recording test.

Reply:
OK, got it~